### DISCUSSION TITLE
[OPERATOR IS DUMB - APOLOGIES] Cannot find type 'TaskPriority' in scope

### DISCUSSION THREAD

**Discussion by technoplato** [pointfreeco/swift-composable-architecture#724] [Q&A] [answered]
Good morning gents. 

I'm fairly new to Swift in general and am really excited to get working with The Composable Architecture.

I'm curious why I would be getting a handful of `TaskPriority` and `Task` errors after adding The Composable Architecture to my project. If relevant - I was attempting to do this on Xcode beta. 

Xcode version: ![Screen Shot 2021-08-19 at 11 22 34 AM](https://user-images.githubusercontent.com/6922904/130095902-0348e23d-22f9-4c4c-b0af-92c1d0dd56a9.png)

Here is my commit: https://github.com/technoplato/FirebaseComposable/commit/4618678045e9ca8b55385f37c1445632f9bff03e

Going to try on regular Xcode in the mean time. 

Thanks for all the great content and thought that went into building this. Very excited to use it!

### Step 1 Attempting to resolve

Opening in non-beta Xcode

![Screen Shot 2021-08-19 at 11 24 08 AM](https://user-images.githubusercontent.com/6922904/130096313-193e16b2-dae4-405a-b1ef-93326cadb360.png)

Looks like this might be how you allow projects from beta Xcode to be opened in older versions

![Screen Shot 2021-08-19 at 11 26 13 AM](https://user-images.githubusercontent.com/6922904/130096598-3c45a52a-4f47-491f-b06c-157c96efa60c.png)

_Non-beta Xcode does not resolve the issue._

### Step 2 - Starting a blank project to test if something is up with my configuration

**Comment by stephencelis** [comment]
@technoplato Looks like you're on Xcode 13 beta 1, but the current version is Xcode 13 beta 5 (13A5212g). A lot has changed in the concurrency APIs since the first beta. Can you download the latest and give things another try?

**Comment by technoplato** [comment, marked as answer]
Thanks so much for such a quick response.

Will give that a try! ☺️
